Amazon.IonObjectMapper.Test/TestObjects.cs

```diff
@@ -1,829 +1,841 @@
-        public void SetLength(int length) 
-        {
-            this.length = length;
-        }
-        
-        [IonPropertyGetter("unit")]
-        public string GetUnit()
-        {
-            return this.unit;
-        }
-
-        [IonPropertySetter("unit")]
-        public void SetUnit(string unit)
-        {
-            this.unit = unit;
-        }
-    }
-
-    public class Chalkboard
-    {
-        public int width { get; set; }
-        public int height { get; set; }
-
-        [IonPropertySetter("width")]
-        public void SetSize(int width, int height)
-        {
-            this.width = width;
-            this.height = height;
-        }
-    }
-
-    public class Country
-    {
-        public string Name { get; init; }
-        public City Capital { get; init; }
-        public Politician President { get; init; }
-        public List<State> States { get; init; }
-    }
-
-    public class State
-    {
-        public string Name { get; init; }
-        public City Capital { get; init; }
-        public Politician Governor { get; init; }
-    }
-
-    public class City
-    {
-        public string Name { get; init; }
-        public Politician Mayor { get; init; }
-    }
-
-    public class Politician
-    {
-        public string FirstName { get; init; }
-        public string LastName { get; init; }
-    }
-
-    public class TestDictionary : Dictionary<string, int>
-    {
-        public static string PrettyString(IDictionary<string, int> dictionary)
-        {
-            return string.Join(Environment.NewLine, dictionary);
-        }
-    }
-
-    public class ClassWithOnlySetProperty
-    {
-        private string val;
-
-        public ClassWithOnlySetProperty(string input)
-        {
-            val = input;
-        }
-
-        public string SetOnlyProperty
-        {
-            set { val = value; }
-        }
-    }
-
-    public class ClassWithProperties
-    {
-        public ClassWithProperties() { }
-
-        public ClassWithProperties(string publicValue, string protectedValue, string protectedInternalValue,
-            string internalValue, string privateValue, string privateProtectedValue)
-        {
-            this.PublicProperty = publicValue;
-            this.ProtectedProperty = protectedValue;
-            this.ProtectedInternalProperty = protectedInternalValue;
-            this.InternalProperty = internalValue;
-            this.PrivateProperty = privateValue;
-            this.PrivateProtectedProperty = privateProtectedValue;
-        }
-
-        public string PublicProperty { get; set; }
-        protected string ProtectedProperty { get; set; }
-        protected internal string ProtectedInternalProperty { get; set; }
-        internal string InternalProperty { get; set; }
-        private string PrivateProperty { get; set; }
-        private protected string PrivateProtectedProperty { get; set; }
-
-        public override string ToString()
-        {
+﻿/*

```

Comment:
Is there a way to fix this diff so it doesn't consider almost the entire file as changed code?

Reply:
The line endings were mismatched with the rest of the project, so that's why it's like this.

Reply:
Were there any noteworthy changes made to this file other than the copyright header and the moved imports?

Reply:
Nope.